services/beeper-connector/UPGRADE.md

```
@@ -0,0 +1,162 @@
+npm run export-rdf        # TypeScript RDF export
+```
+
+## Architecture Changes
+
+### v1.0 (Python)
+```
+Beeper DB → Python Script → RDF File → Manual Import
+```
+
+### v2.0 (TypeScript)
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)